views/courses/components/template-new-course-time.hbs

```diff
@@ -18,7 +19,14 @@
         <input id="startOfLesson" class="form-control new-course-time-template-startTime" data-time type="text"/>
     </td>
     <td>
-        <input id="lengthOfLesson" class="form-control new-course-time-template-duration" type="number" placeholder="{{$t "courses.global.input.inMinutes"}}" min="0" />
+        <div>
+            <input id="lengthOfLesson" class="form-control new-course-time-template-duration lengthOfLesson" type="number" placeholder="{{$t "courses.global.input.inMinutes"}}" min="0" />
+        </div>
+        <div>
+          <span class="small" id="courseTimesErr" style="visibility:hidden; color: #ff1134;">
```

Comment:
why do you use color css here instead of using classes?

Reply:
done